pastas/stressmodels.py

```diff
@@ -97,6 +105,11 @@ def __init__(
 
         self.stress = []
 
+        if CACHETOOLS_AVAILABLE:
+            self._cache = LRUCache(maxsize=max_cache_size)
```

Comment:
so here you have self._cache = pastas.decorators.get_cacher()
```
def get_cacher(max_cache_size: int) -> Cacher | None:
    try: 
        import ...
    except:    
        ...
```

Then we can potentially use it in other places too right?